Doc/c-api/set.rst

```diff
@@ -147,7 +147,7 @@ subtypes but not for instances of :class:`frozenset` or its subtypes.
 
    Return ``1`` if found and removed, ``0`` if not found (no action taken), and ``-1`` if an
    error is encountered.  Does not raise :exc:`KeyError` for missing keys.  Raise a
-   :exc:`TypeError` if the *key* is unhashable.  Unlike the Python :meth:`~set.discard`
+   :exc:`TypeError` if the *key* is unhashable.  Unlike the Python :meth:`~frozenset.discard`
```

Comment:
Ideally we'd fix it so that the original linked properly, given that frozensets don't have a `.discard()` method, but sets do 🙃

Reply:
Good idea, although I'm not sure how to do it. PR welcome! 🙃